frontend/ui/data/css_tweaks.lua

```diff
@@ -1045,6 +978,21 @@ This tweak can be duplicated as a user style tweak when books contain footnotes
         (function()
             local sub_table = {
                 title = _("In-page footnote font size"),
+                {
+                    id = "inpage_footnote_font-size_smaller",
+                    conflicts_with = function(id) return util.stringStartsWith(id, "inpage_footnote_font-size_") end,
+                    title = _("Smaller footnotes (80%, legacy)"),
+                    description = _([[
+Decrease size of in-page footnotes. This may have limited/no apparent effect, e.g. if it's overwritten by publisher styles. The "footnote font size" style tweaks are recommended instead.]]),
```

Comment:
```suggestion
Decrease size of in-page footnotes. This may have no effect if it's overwritten by publisher styles. The "footnote font size" style tweaks are recommended instead.]]),
```